### DISCUSSION TITLE
Custom  snapshotFileUrl

### DISCUSSION THREAD

**Discussion by SteinerHannes** [pointfreeco/swift-snapshot-testing#497] [Ideas]
```
let snapshotFileUrl = snapshotDirectoryUrl
        .appendingPathComponent("\(testName).\(identifier)")
        .appendingPathExtension(snapshotting.pathExtension ?? "")
```
The current implementation (see above) does not allow fully customizable file names. 

@stephencelis @mbrandonw is there a reason why?

**Comment by mbrandonw** [comment]
Can you explain more about what you mean by "customizable file names"?

**Reply by SteinerHannes** [reply]
For example, I don't want a period in the name. 

Currently I am limited to `<something>.<something else>`

**Reply by SteinerHannes** [reply]
Do you unterstand what I mean?
